testsuite/domain/src/test/resources/host-configs/host-slave-rbac-properties.xml

```diff
@@ -81,7 +81,7 @@
 
     <domain-controller>
         <!-- Remote domain controller configuration with a host and port -->
-        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm">
+        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm" ignore-unused-configuration="false">
```

Comment:
FullRbacProviderPropertiesRoleMappingTestSuite passes without this change.

This file includes servers from each group in domain-rbac.xml, so even with ignoring all data should be present; i.e. true or false should make no difference. So 'true' or undefined is a better test since it may pick up regressions.